contracts/VerifierRegistry.sol

```diff
@@ -1,10 +1,24 @@
-pragma solidity ^0.5.0;
+pragma solidity 0.5.0;
 
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
-import "token-sale-contracts/contracts/Token.sol";
-import "token-sale-contracts/contracts/HumanStandardToken.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import "contract-registry/contracts/interfaces/IContractRegistry.sol";
+import "contract-registry/contracts/storage/interfaces/IStorageBase.sol";
+import "contract-registry/contracts/storageStrategy/interfaces/IStorageStrategy.sol";
+import "contract-registry/contracts/interfaces/RegistrableWithSingleStorage.sol";
+import "token-incentivized-sidechains/contracts/interfaces/IStakingBank.sol";
+
+import "./interfaces/IVerifierRegistry.sol";
+import "./VerifierRegistryStorage.sol";
+
+contract VerifierRegistry is IVerifierRegistry, Ownable, RegistrableWithSingleStorage {
+
+  using SafeMath for uint256;
+
+  // 0x5665726966696572526567697374727900000000000000000000000000000000
```

Comment:
what does the address mean?

Reply:
this is not the address, this is name representation in bytes32